src/styles/components/_code-block.scss

```diff
@@ -7,7 +7,7 @@ $breakpoint-increase-padding: $breakpoint-scale-up;
   background-color: var(--background-color);
 
   &__heading {
-    font-size: 1.5625rem; // 25px/16
+    font-size: map.get($font-sizes, "mobile-heading"); // 28px/16
```

Comment:
So throughout this PR, I think we should delete the comments indicating pixel sizes anywhere that we're using the `map.get` to get a size from our `$font-sizes`.

The reason being that these comments are lies just waiting to happen. If the `mobile-heading` size changes (which it did), then all of these comments are incorrect.

We can leave the comments in place within the `$font-sizes` map, though.

Reply:
That makes perfect sense. They should all be deleted now: https://github.com/sparkbox/accessible-components/pull/118/commits/071fe42b817aa13e5ec107eeb615f3930313ef5e

Reply:
"lies waiting to happen--that's perfect!" :D